geode-protobuf/src/main/proto/v1/clientProtocol.proto

```diff
@@ -35,7 +35,6 @@ message Message {
 }
 
 message Request {
-    CallbackArguments callbackArg = 1;
     oneof requestAPI {
         PutRequest putRequest = 2;
```

Comment:
Since callback arg is gone, should putRequest actually start with 1?

Reply:
I kind of want to keep the one available just in case, but I don't think it's actually a good idea. Probably we should decrement all the indices.

Reply:
I decided to address this by changing the requestAPI field numbers to start at 10.  This lets us add some fields to these messages in the future without having to worry about colliding with the requestAPI field numbers (as we do expect to keep adding new types of messages here).